src/components/RunningTrackComponent.vue

```diff
@@ -199,7 +199,12 @@ export default defineComponent({
       }
     },
     sortedDrawingOrder(teams: Array<Team> | undefined) {
-      return [...teams || []].sort((a, b) => a.rounds - b.rounds);
+      return [...teams || []].sort((a, b) => {
+        const diff = a.rounds - b.rounds
+        if (diff !== 0) return diff;
+
+        return a.updatedAt - b.updatedAt;
```

Comment:
Wouldn't it make more sense to compare the progress [over the current round] if they have the same amount of rounds?

Reply:
Yeah definitely